portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,22 +1,24 @@
 package com.google.sps.data;
 
 import java.util.Date;
+import java.util.List;
 
 public class Comment {
     private long id;
     private String name;
     private String description;
     private long postDate;
+    private List<String> images;
```

Comment:
Should images be something like a ByteString? If it is a String it should have a comment explaining why it is

Reply:
On second pass this should be imageUrls